flocker/node/agents/functional/test_testtools.py

```diff
@@ -1,20 +0,0 @@
-# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
-
-"""
-Tests for ``flocker.node.agents.testtools``.
-"""
-
-from ..testtools import (
-    make_icindervolumemanager_tests, tidy_cinder_client_for_test
-)
-
-
-class TidyCinderVolumeManagerInterfaceTests(
```

Comment:
Should these tests be deleted, or do we still want to test the underlying implementation?

If we don't want to keep them, `make_icindervolumenanager_tests` can also be removed.